stdlib/public/Concurrency/Actor.cpp

```diff
@@ -387,6 +405,32 @@ static void checkIsCurrentExecutorMode(void *context) {
                                         : Swift6_UseCheckIsolated_AllowCrash;
 }
 
+#if defined(__APPLE__)
+#define SWIFT_LOG_APPLE_RUNTIME_ISSUES_SUBSYSTEM "com.apple.runtime-issues"
+#define SWIFT_LOG_ACTOR_CATEGORY "Actor"
+os_log_t IsolationWarningLog;
+#endif
+
+static void initIsolationWarningOsLog(void *context) {
+#if defined(__APPLE__)
+  IsolationWarningLog = os_log_create(SWIFT_LOG_APPLE_RUNTIME_ISSUES_SUBSYSTEM,
+                                      SWIFT_LOG_ACTOR_CATEGORY);
+#endif
+}
+
+/// Forces stderr logging of isolation warnings,
+/// in addition to the os_log warning issued on apple platforms.
+bool LogIsolationWarningStderr = false;
+static void initLogIsolationWarningStderr(void *context) {
```

Comment:
This is primarily here so we can test stderr in our tests. Or on Linux if someone wanted to get warnings.

TODO: actually perhaps enable this by default on linux...?